lxd/container_test.go

```diff
@@ -141,12 +142,7 @@ func (suite *containerTestSuite) TestContainer_LoadFromDB() {
 	_, err = c2.StorageStart()
 	suite.Req.Nil(err)
 
-	// When loading from DB, we won't have a full LXC config
-	c.(*instanceDrivers.LXC).c = nil
-	c.(*instanceDrivers.LXC).cConfig = false
-	c2.(*instanceDrivers.LXC).c = nil
-	c2.(*instanceDrivers.LXC).cConfig = false
-
+	instanceDrivers.PrepareEqualTest(c, c2)
```

Comment:
Is that function only exported from a `_test.go` too?

Reply:
See https://github.com/lxc/lxd/pull/6954#discussion_r385948920

Reply:
@stgraber according to https://github.com/golang/go/issues/10184 a function defined in a package's test file and then imported won't be available, as test files are not included in the normal build. Also using reflect to modify, rather than access an unexported variable, especially setting it to nil, seems to be rather messy, requiring the use of the `unsafe` package. So for a readability perspective I'd like to keep a function in the `instance/drivers` pkg, although the name/structure could change.

For example I could n `instance/drivers/test_util.go` file with this function in instead to make it clear its for tests?

Reply:
Yeah, sounds reasonable